libnetwork/netavark/config.go

```diff
@@ -326,6 +326,11 @@ func createIpvlanOrMacvlan(network *types.Network) error {
 					return fmt.Errorf("unknown ipvlan mode %q", value)
 				}
 			}
+		case types.MetricOption:
+			_, err := strconv.ParseUint(value, 10, 32)
```

Comment:
Nit: if _, err := ...

Reply:
I am following the style of the rest of the option parsing code with that so I have a slight preference to keep it like this.